cmd/bundle/generate/generate_test.go

```diff
@@ -215,6 +216,73 @@ func TestGenerateJobCommand(t *testing.T) {
 	require.Equal(t, "# Databricks notebook source\nNotebook content", string(data))
 }
 
+func TestGenerateJobWithGitSource(t *testing.T) {
```

Comment:
Can this be an acceptance test?

Reply:
It could, I'm currently exploring this or integration test as we have one for generate already

Reply:
IMO acceptance tests are a better fit here since we compare the generated files. You can also choose to run them as integration tests by not setting `LocalOnly = true`

Reply:
Added acceptance test